### PR TITLE
Update optional-lite to 3.3.0

### PR DESCRIPTION
**Pull request by Minoru**
The code was adjusted to avoid `maybe-unitialized` warnings on i686:
https://cirrus-ci.com/task/4915794935021568?command=build#L68

### CODE REVIEW

File: src/feedlistformaction.cpp [hunk 1]
```diff
@@ -273,7 +273,8 @@ bool FeedListFormAction::process_operation(Operation op,
 					v->show_error(_("Failed to spawn browser"));
 					return false;
 				} else if (*exit_code != 0) {
```
Comment (dennisschagt) [inline src/feedlistformaction.cpp:275]: Nitpickky suggestion for consistency:
```suggestion
				} else if (exit_code.value() != 0) {
```

**Review by dennisschagt — APPROVED** [review]
:+1:

File: src/feedlistformaction.cpp (hunk 1 above)
Comment (Minoru) [reply src/feedlistformaction.cpp]: Yeah, makes sense, thank you!

**Comment by Minoru** [comment]
Thanks for the review, @dennisschagt!

**Pull request merged**